lib/features/pokemon_details/pokemon_details_page.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:pokedex/features/pokemon_details/pokemon_about/pokemon_about_page.dart';
+import 'package:pokedex/features/pokemon_details/pokemon_moves/pokemon_moves_page.dart';
+import 'package:pokedex/features/pokemon_details/pokemon_stats/pokemon_stats_page.dart';
+import 'package:pokedex/features/pokemon_overview/pokemon_overview_connector.dart';
+import 'package:pokedex/utils/strings.dart' as str;
+
+class PokemonDetailsPage extends StatelessWidget {
+  const PokemonDetailsPage({
+    Key? key,
+  }) : super(key: key);
+
+  static const String route = '/details';
```

Comment:
for sub route go_router won't allow a route that starts/ends with `/` - it's intended for absolute route 🙂